generators/app/templates/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "<%=testProjectName%>",
+  "name": "myproject",
```

Comment:
Please revert this. Yeoman Generator will put project name here, depending on what was passed as parameters